source/plugin/project/version-numbers.rst

```diff
@@ -0,0 +1,82 @@
+Version Numbers
+===============
+
+Your plugin needs a unique version number in order to allow humans to find out which version of a plugin is the
+successor of the other. There are plenty of versioning schemes that you could use for your own plugin. In the following
+part we explain a few concepts with their pros and cons.
+
+Sponge recommends to use a version identifier that consists of either two or three numeric version parts separated with
+a ``.`` and additional label appended with a hyphens ``-`` such as ``3.1`` or ``1.0.3-API7`` similar to SemVer.
+Start with ``1.x`` if your plugin is ready. Start with ``0.x`` if your plugin is not yet ready or unstable.
+
+.. note::
+
+    While Sponge recommends using the above versioning scheme, plugin authors are free to choose their own. Please keep
+    in mind that your users have to deal with a multitude of versioning schemes, so keep it simple.
+
+SemVer
+------
+
+`SemVer <https://semver.org/>`_ is a commonly used semantic versioning scheme. It consists of three numeric blocks that
+are separated by dots and additional labels which are appended using hyphens. The three numeric blocks make up
+the ``MAJOR``, ``MINOR`` and ``PATCH`` version parts. 
+
+* A change in the **MAJOR** version indicates that you made incompatible API changes. Rewriting large parts of your
+  plugin or the entire plugin would also count as incompatible changes.
+* A change in **MINOR** version indicates that a feature has been added without breaking backwards
+  compatibility.
+* A change in the **PATCH** version indicates that some bugs have been fixed.
+
+The labels can be used to mark builds as pre-releases or contain build metadata such as the supported minecraft version.
+If you increase one part of the version, then you will set the following parts back to zero. Using this strategy you
+can find the latest version by comparing the version blocks with each other, however this scheme does not contain any
```

Comment:
It's a bit unclear what the part after "however" means. An example might help.

Reply:
Fixed